packages/server-renderer/src/renderToString.ts

```diff
@@ -177,7 +177,8 @@ function renderComponentSubTree(
   if (isFunction(comp)) {
     renderVNode(push, renderComponentRoot(instance), instance)
   } else {
-    if (!comp.ssrRender && !comp.render && isString(comp.template)) {
+    if (!comp.ssrRender && !instance.render && isString(comp.template)) {
+      // TODO: remove backported fix
```

Comment:
These fixes are backports of #720 required for tests to pass. Now this PR is locked or merge until #720 will be merged into `master`, but open for any reviews or improvements.